doc-content/enterprise-only/openshift/registry-auth-proc.adoc

```diff
@@ -0,0 +1,30 @@
+[id='registry-auth-proc']
+= Configuring OpenShift registry authentication
+
+You must configure OpenShift registry authentication to access Red Hat product images. 
+
+This procedure is not required if you are using a Red Hat OpenShift Online account.
+
+.Procedure
+
+. Complete the steps documented in https://access.redhat.com/RegistryAuthentication#registry-service-accounts-for-shared-environments-4[Registry Service Accounts for Shared Environments]. You must log on to Red Hat Customer Portal to access the document and to complete the steps. You must create a registry service account.
+
+. Select the *OpenShift Secret* tab and click the link under *Download secret* to download the YAML secret file.
+
+. View the downloaded file and note the name that is listed in the `name:` entry.
+
+. Run the following commands:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create -f <file_name>.yaml 
+oc secrets link default <secret_name> --for=pull
+oc secrets link builder <secret_name> --for=pull
+----
++
+Where `<file_name>` is the name of the downloaded file and <secret_name> is the name that is listed in the `name:` entry of the file. 
+
+[NOTE]
+===
```

Comment:
Generated doc contains "===" in note section.
I suppose there needs to be one more equal sign - "===="